operators/src/adapters/raster_subquery/raster_subquery_reprojection.rs

```diff
@@ -144,6 +116,98 @@ where
     }
 }
 
+fn projected_coordinate_grid_parallel(
+    pool: &ThreadPool,
+    tile_info: TileInformation,
+    out_srs: SpatialReference,
+    in_srs: SpatialReference,
+    valid_out_area: &SpatialPartition2D,
+) -> Result<Grid2D<Option<Coordinate2D>>> {
+    const OVERPROVISIONING: usize = 1; // split the work into multiple chunks per thread.
+    const MAX_PARALELLISM: usize = 8; // max task / work split
```

Comment:
Is it better to have a max parallelism or a minimum chunk size?
Maybe the latter one is more robust when the chunk size would change.

Reply:
changed that

Reply:
7508587